docs/provisioning-methods/zte.md

```diff
@@ -0,0 +1,76 @@
+# How to Onboard a Device Using ZTE?
+
+Zero-touch Enrollment (ZTE) is an onboarding method offered by Google for the [selected devices](https://androidenterprisepartners.withgoogle.com/devices). Upon first boot or factory reset, ZTE forces Android devices to enroll into Esper. Follow the steps below to onboard a device using ZTE.
+
+:::tip
+This is an alternative method for enrolling devices to the Console.
+::: 
+
+Step 1: Navigate and log in to the [Zero-touch Enrollment (ZTE) portal](https://partner.android.com/zerotouch).
+
+Step 2: You will be taken to the Configuration tab.
+
+  
+
+![portal](./images/ZTE/1-ZTE.png)
+
+Here you will be able to create a configuration to associate with a device.
+
+Step 3: Click the ‘+’ symbol to add a new configuration.
+
+Step 4: On the pop-up, enter the details— configuration name and EMM DPC (Esper Device Management). Enter your organization’s information for the company name, support email address, support phone number.
+
+![create config](./images/ZTE/2-createConfig.png)
+
+To enter the DPC extras, navigate to the Esper Console.
+
+![download](./images/ZTE/3-downloadCOnfig.png)
+
+Login to the Esper Console and navigate to the Provisioning Template section. Click 'View QR code' link associated with the Template you want to configure with ZTE and click the “Download Config” button at the bottom of the pop-up window with your QR code. Open the downloaded JSON config file, and you will need to copy the following contents: 
+
+**Note:** In the downloaded config file, there will be a section of the code that is not needed. Copy only the section below, and use a formatter such as [JSON formatter](https://jsonformatter.curiousconcept.com/) to ensure the code is correctly formatted.
+
+```json
```

Comment:
@radhak21 please format the code yourself as well, the spaces and alignment as of now is not correct